CLAUDE.md

```
@@ -49,3 +49,34 @@ Third-party plugins keep their own `repo` URL and empty `plugin_path`.
+
+**Before opening a PR that changes a plugin:**
+```bash
+# from a LEDMatrix (core) checkout, with the monorepo plugins on the path:
+python scripts/check_plugin.py --plugin <id> \
+  --plugin-dir /path/to/ledmatrix-plugins/plugins --out-dir /tmp/preview
+```
+Eyeball the PNGs in `/tmp/preview`, then fix any FAIL (overflow/crash) before pushing.
+
+**Golden images (optional, per plugin):** commit reference PNGs so visual drift is
+caught automatically:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)